sdk/liftrqumulo/Azure.ResourceManager.Qumulo/src/Generated/AgentResource.cs

```diff
@@ -0,0 +1,242 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Globalization;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Qumulo.Models;
+
+namespace Azure.ResourceManager.Qumulo
+{
+    /// <summary>
+    /// A Class representing an Agent along with the instance operations that can be performed on it.
+    /// If you have a <see cref="ResourceIdentifier" /> you can construct an <see cref="AgentResource" />
+    /// from an instance of <see cref="ArmClient" /> using the GetAgentResource method.
+    /// Otherwise you can get one from its parent resource <see cref="StorageMoverResource" /> using the GetAgent method.
+    /// </summary>
+    public partial class AgentResource : ArmResource
```

Comment:
The name is too generic. Consider appending RP prefix or renaming it.
- use `prepend-rp-prefix` to automatically append PR name. See example [here](https://github.com/Azure/azure-sdk-for-net/blob/176fd476588e8e61ebce747d66714cb39947cae5/sdk/azurestackhci/Azure.ResourceManager.Hci/src/autorest.md?plain=1#L52-L57).
- use `rename-mapping` to rename a model. See example [here](https://github.com/Azure/azure-sdk-for-net/blob/176fd476588e8e61ebce747d66714cb39947cae5/sdk/azurestackhci/Azure.ResourceManager.Hci/src/autorest.md?plain=1#L58-L76)

Reply:
@archerzz we have updated the name by appending Qumulo in these classes. please review.